src/components/MapBox.tsx

```diff
@@ -67,17 +65,6 @@ const MapSettings: React.FC<{
     logger.debug(`zoom: ${zoom}`);
   }, [zoom]);
 
-  useEffect(() => {
-    const h3 = geoToH3(center[0], center[1], utils.constants.DefaultH3Resolution);
-    const h3Indexes = kRing(h3, utils.constants.DefaultRingSize);
-
-    h3Indexes.forEach((h) => {
-      L.polygon(h3ToGeoBoundary(h) as unknown as LatLngExpression[][], {
```

Comment:
It is very challenging for me to understand what is happening.

Is there a way to make it more declarative?

Reply:
that was hexagons on map (not required for new implementation)